docs/content/docs/operations/metrics-logging.md

```diff
@@ -63,7 +56,7 @@ In addition to the simple counts we further track a few selected state transitio
  - Rollback : Time from deployed to rolled_back state if the resource was rolled back
  - Submission: Flink resource submission time
 
-## Kubernetes Client Metrics
+### Kubernetes Client Metrics
 
 The Operator gathers various metrics related to Kubernetes API server access. The Kubernetes client metrics can be enabled by the configuration `kubernetes.operator.kubernetes.client.metrics.enabled` (default: `true`).
```

Comment:
We could probably remove the property from here. (or add it everywhere :)

Reply:
I added a general note and a link to the general config section